style.css

```diff
@@ -66,3 +64,54 @@ body {
   height: 342px;
   margin: 15px 0px 17px;
 }
+.fa {
+  font-size: 30px;
+  width: 30px;
+  text-align: center;
+  text-decoration: none;
+  padding: 10px;
+}
+.fa:hover {
+  background-color: var(--items-bg-color);
+  border-radius: 50%;
+}
+.fa-twitter {
+  color: white;
+}
+.nav-menu-list{
+  list-style-type: none;
+  padding: 0px;
+  margin-top:0px;
+}
+.nav-items{
+  padding: 10px 0 0;
+  margin-top: 0px;
+}
+.fa-icon{
+  font-size: 25px;
+  width: 10px;
+  text-decoration: none;
+  padding: 10px;
+  color: var(--text-color);
+}
+.fa-icon:hover{
+  background-color: var(--items-bg-color);
+  border-radius: 25px;
+}
+.tweet-button{
+  background-color: var(--tweet-button-color);
+  border: none;
+   border-radius: 25px;
+   font-size: 16px;
+   width: 100%;
+   color: var(--text-color);
+   padding: 16px;
+   font-weight: bold;
+}
+.tweet-button:hover{
+  opacity: 0.7;
+}
+.nav-item{
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
I think you should add this `font-family` once on the body styling and not add it again in any element except if the element has a different one